docs/index.md

```diff
@@ -181,7 +178,7 @@ class TestCrud(unittest.TestCase):
   - Then, it retrieves the user using `self.your_core_lib.user.get()` method with the user's ID obtained from the creation step.
   - Finally, it asserts that the retrieved user matches the created user.
 
-##### **your_core_lib_instance.py**
+### `your_core_lib_instance.py`
 
 ```python
```

Comment:
let fix this 
Implementing a singleton pattern for your YourCoreLib class using a separate class YourCoreLibInstanc

to something LIKE
Implementing a singleton pattern to hold your YourCoreLib class for using it when needed in multiple web views, same core-lib instance for multiple test files